extensions/cornerstone/src/Viewport/Overlays/ViewportOrientationMarkers.tsx

```diff
@@ -118,6 +118,10 @@ function ViewportOrientationMarkers({
       viewportIndex
     );
 
+    if (!ohifViewport) {
+      console.log('No viewport');
```

Comment:
No viewport is not enough, maybe say `console.log(ViewportOrientationMarkers::No viewerport)` or something more